hidtest/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+project(hidtest C)
+
+set(HIDAPI_HIDTEST_TARGETS)
+if(NOT WIN32 AND NOT APPLE AND CMAKE_SYSTEM_NAME MATCHES "Linux")
```

Comment:
```suggestion
if(UNIX AND NOT APPLE)
```

Reply:
it is not the same, this is really means "Linux only, and no other UNIX system"